x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/index.tsx

```diff
@@ -295,6 +321,7 @@ export const QueryTabContentComponent: React.FC<Props> = ({
                 className="timeline-flyout-body"
               >
                 <StatefulBody
+                  defaultModel={timelineDefaults}
```

Comment:
I think you wanted to remove that as well

Reply:
yeah but if we remove it here, then it couldn't get the right fallback value in `x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.tsx` Line 215

Reply:
that happens in cypress test, but seems to be fine in ordinary env.

Reply:
what is the Cypress error?

Reply:
it couldn't get the right value of `columns` or sometime missing `showCheckboxes` on detections page
![Screenshot 2020-12-11 at 18 26 33](https://user-images.githubusercontent.com/6295984/101892574-78f19500-3b9b-11eb-8f2b-9ff5e6d59488.png)